examples/www-image/main/bmp-render.c

```diff
@@ -0,0 +1,592 @@
+/* Simple firmware for a ESP32 displaying a static image on an EPaper Screen */
```

Comment:
This seems not functional ATM. I didn't see any epd_poweron()? 
Do you still want to include the bitmap version? It may be easier to maintain just the jpg vesion as an example.

Reply:
I forgot to add it. But is interesting to see that refreshes and works even without calling this.